tests/integration/users/test_views.py

```diff
@@ -0,0 +1,86 @@
+import json
+
+from openslides.utils.test import TestCase
+
+
+class TestWhoAmIView(TestCase):
+    url = '/users/whoami/'
+
+    def test_get_anonymous(self):
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content, b'{"user_id": null}')
```

Comment:
Using 'data' is recommended by the Django RESt Framework: http://www.django-rest-framework.org/api-guide/testing/#testing-responses